software/ollama.yml

```
@@ -0,0 +1,12 @@
+name: "Ollama"
+website_url: "https://ollama.com/"
+source_code_url: "https://github.com/ollama/ollama"
+description: "Get up and running with Llama 3.3, DeepSeek-R1, Phi-4, Gemma 3, and other large language models."
+licenses:
+  - MIT License
+platforms:
+  - Docker
+  - Python
+tags:
+  - Software Development - API Management
```

```suggestion
  - Software Development - API Management
```

No longer relevant since the introduction of the `Generative Artificial Intelligence (GenAI)` tag